README.md

```diff
@@ -316,16 +316,17 @@ yarn test
 
 ### Releasing
 
-For ease the release process, we use `release-it` utility that helps with the process.
+For ease the release process, we use `release-it` utility that helps with the process and `towncrier` for creating and maintaining changelogs.
 
 https://www.npmjs.com/package/release-it
+https://towncrier.readthedocs.io/
 
 For using it and start a release you need to fulfill the requirements:
 
 - Have permissions to push on master branch
```

Comment:
What do I need do to request such permission?

```suggestion
- Have permission to push to `master` branch.
```

Reply:
If one wants to release Volto, he might already have, right? I mean, it would have taken the required actions to have permissions.

Reply:
I cannot inspect the settings of the volto repo to answer that. That's why I'm asking. :wink:

I know that I need to have signed the Plone Contributor Agreement.

I know I have write permission to create a new branch in `plone/volto`, but why? Is it because a group of which I am a member has write permission?

For `master`, is it restricted in any way beside CI checks? Again, I know I can push to `master`, but what gives me that permission?